contributors/devel/sig-node/e2e-node-tests.md

```diff
@@ -16,8 +16,8 @@ Prerequisites:
 - [Install etcd](https://github.com/coreos/etcd/releases) and include the path to the installation in your PATH
   - Verify etcd is installed correctly by running `which etcd`
   - Or make etcd binary available and executable at `/tmp/etcd`
-- [Install ginkgo](https://github.com/onsi/ginkgo) and include the path to the installation in your PATH
```

Comment:
was it changed? I still install ginkgo

as per CNI - I guess it is configured on my dev machine. When your are trying on clean machine - what does "working" mean? Can you please clarify or maybe add a link?

Reply:
The Makefile [dependency](https://github.com/kubernetes/kubernetes/blob/ae23b0c11bc861bdca346666a739560ea3ee3145/build/root/Makefile#L272) on ginkgo which installs it has been there for a while.  It then uses that version to run the tests [here](https://github.com/kubernetes/kubernetes/blob/ae23b0c11bc861bdca346666a739560ea3ee3145/test/e2e_node/runner/local/run_local.go#L59). I looked at the remote code, and though I haven't tested it, it appears to copy the built version of [ginkgo](https://github.com/kubernetes/kubernetes/blob/ae23b0c11bc861bdca346666a739560ea3ee3145/test/e2e_node/remote/node_e2e.go#L63) along with kubelet, etc.  I tested on a new machine this morning and e2e node tests ran without manually installing ginkgo.

Yup, will update the CNI info.